shared-functions

```diff
@@ -92,15 +92,30 @@ function bosh_interpolate() {
     arguments="${arguments} -l vars-files/${vf}"
   done
 
-  if [ -n "${release_name}" ]; then
+  if [ -z "${RELEASE_TARBALL}" ]; then
```

Comment:
we check if the pipeline has set a `RELEASE_TARBALL` env var and then go to a flow that we used to have. If there is a better way, suggestions are welcomed.